src/wp-includes/block-template-utils.php

```diff
@@ -894,6 +894,14 @@ function _build_block_template_result_from_post( $post ) {
 		}
 	}
 
+	$hooked_blocks = get_hooked_blocks();
```

Comment:
If we leave out the change in this file, we could land the rest at any point. So the question is whether we can think of any potential issue with making the edited templates hookable?

Do we have user-created patterns covered?

Reply:
> If we leave out the change in this file, we could land the rest at any point. So the question is whether we can think of any potential issue with making the edited templates hookable?

We can give it some smoke testing, plus maybe some more unit test coverage (e.g. in `tests/phpunit/tests/block-templates/buildBlockTemplateResultFromPost.php`) for increased confidence. 

> Do we have user-created patterns covered?

I haven't checked yet! I'm assuming that yes (since AFAIK their only "entry point" is the Patterns Registry class), but it'll be good to verify :+1:

Reply:
I think that leaves only Synced Blocks (aka Reusable Blocks), which are loaded through code similar to:

```php
$reusable_block = get_post( $attributes['ref'] );
$content = do_blocks( $reusable_block->post_content );
```